R/glimpse.R

```diff
@@ -128,6 +128,10 @@ format_glimpse <- function(x, ...) {
 
 # A variant with collapse and without checks, for format_glimpse.list()
 format_glimpse_1 <- function(x, ...) {
+  paste0(format_glimpse_(x, ...), collapse = crayon_grey_0.6(", "))
```

Comment:
What's the reason we don't use `collapse()` here?

Reply:
`collapse()` is `paste(x, collapse = ", ")`, but we need the arg `collapse = crayon_grey_0.6(", ")`